hasher-matcher-actioner/requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
 pytest==6.2.1
 mypy==0.812
+moto
```

Comment:
You will have to figure out how to do this once you try to merge into master. David's setup.py changes will need to be accounted for. But you already knew that. :)

Reply:
I think we only need to worry about setup.py for deployed images, if you run the commands that are in the CI locally this seems to be enough but I think it is probably work consolidating to just using setup.py eventually